app/assets/javascripts/darkswarm/directives/darker_background.js.coffee

```diff
@@ -0,0 +1,4 @@
+Darkswarm.directive "darkerBackground", ->
+  restrict: "A"
+  link: (scope, elm, attr)->
+    elm.closest('.page-view').toggleClass("with-darker-background", true)
```

Comment:
so this directive makes the background color toggleable? I'm not sure I'm following.

Reply:
yes, that's it, it toggles the background color of the parent page-view element. This avoids having to repeat the conditionals when we define the parent element on the server (i.e. if tab is shop and shop is closed OR...)

Reply:
If there's any other way we can set this background colour with just CSS (without adding this extra directive) I think I'd favour that. I'll have a quick look at it now...

Reply:
Ok, it's pretty tricky. I think it can be done, but it might take a lot of refactoring and moving everything around :see_no_evil: 

I guess this option is simpler :+1:

Reply:
yes, I tried the other way with conditionals on the server side! I then moved to this one and thought this was better.